source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,284 @@
+
+Select an incident to view:
+
+- Incident name
+- A link icon to open the corresponding incident channel
+- Status (**Ongoing** or **Ended**)
+- The Commander, including profile picture and username
+- A prompt to export the incident channel
+- The total duration
+- The number of members involved in the incident
+- The number of messages posted to the channel, including another prompt to open the corresponding incident channel
+
+The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it
+shows the end time (in the user's timezone). The **Members Involved** widget indicates the total number of users that participated in the channel, either
+by posting a message, being assigned as Commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
+
+The total number of messages displayed includes messages posted by both users and bots (including the incident response bot). It does not include system or ephemeral messages.
+
+**Filtering incidents**
+
+Incidents can be filtered by incident name, Commander, and incident status.
+
+**Browsing related channels**
+
+Incident members see a link to the incident channel at the bottom of the incident details. Clicking the channel name navigates to the incident channel.
+This section is omitted when the active user is not an incident member.
+
+Exporting channels
+^^^^^^^^^^^^^^^^^
+
+If your server is licensed for E20, and the channel export plugin is installed and active, navigate to **Main Menu > Incidents & Playbooks**, select an incident, and
+then choose **Export Incident Channel** in the top-right corner to download the contents of the incident channel as a CSV. The file excludes attachments, but includes system messages.
+If you have an E20 license but the channel export plugin is not installed, or the plugin is installed but not enabled, it’s not possible to select **Export Incident Channel**.
+
+To install and activate the plugin, navigate to the plugins menu and follow the steps provided.
+
+Telemetry
+^^^^^^^^^^
+
+During beta early access, events for the Incident Response plugin are collected regardless of the server telemetry configuration. In other words, even if
+telemetry is disabled in your Mattermost server, the information described on this page is still collected.
+
+We only track the events that create, delete or update any items. We never track the specific content of the items. In particular, we
+do not collect the name of the incidents or the contents of the checklist items.
+
+Every event we track is accompanied with metadata that help us identify each event and isolate it from the rest of the servers. We can group all
+events that are coming from a single server, but never identify that server. The following list details the metadata that accompanies every event:
```

Comment:
I need to change this in the Confluence page, I realized it is not true. It will get published with the rest of the changes for 0.4

Reply:
I've updated the doc with the new content, thanks! :)